Exiled.API/Features/DamageHandler.cs

```diff
@@ -91,13 +91,17 @@ public DamageType Type
                         case WarheadDamageHandler _:
                             return DamageType.Warhead;
                         case ScpDamageHandler scp:
-                            return scp._translationId == DeathTranslations.PocketDecay.Id ? DamageType.PocketDimension : DamageType.Scp;
+                            return DamageType.Scp;
                         case ExplosionDamageHandler _:
                             return DamageType.Explosion;
                         case Scp018DamageHandler _:
                             return DamageType.Scp018;
                         case RecontainmentDamageHandler _:
                             return DamageType.Recontainment;
+                        case CustomReasonDamageHandler_:
+                            return DamageType.CustomDamage;
+                        case Scp096DamageHandler _:
```

Comment:
I don't see why Scp096 needs his own damage type, this should return DamageType.Scp instead.

Reply:
is required an 
DamageType.Scp096
Because he use 
Scp096DamageHandler
And not
ScpDamageHandlers

Reply:
Yes so simple making Scp096DamageHandler return DamageType.Scp aswell.
He doesn't need his own DamageType enum value.